src/main/resources/application.yml

```diff
@@ -25,3 +25,19 @@ ribbon:
   eureka:
     enabled: true
 
+
+#Resilience4j config: Have to run some tests to figure out best config
```

Comment:
AS mentioned in the comment, going to want to test what config is best for our system in the future. This is just for now